docs/_docs/usegotemplates.md

```diff
@@ -86,13 +86,13 @@ The content of `tables.md`:
 
 This is how the OpenAPI file would be rendered in [Swagger UI](https://swagger.io/tools/swagger-ui/)
 
-![Screenshot swaggerfile in SwaggerUI](https://raw.githubusercontent.com/grpc-ecosystem/grpc-gateway/master/docs/_imgs/gotemplates/swaggerui.png)
+![Screenshot OpenAPIfile in SwaggerUI](https://raw.githubusercontent.com/grpc-ecosystem/grpc-gateway/master/docs/_imgs/gotemplates/swaggerui.png)
```

Comment:
Lets make this `OpenAPI file`, I don't think they're often concatenated like this unlike `swaggerfile`.

Reply:
Sure, that make sense. Fixed now.